### DISCUSSION TITLE
Training outdoor

### DISCUSSION THREAD

**Discussion by goback1** [community/community#36255] [Mobile]
How to get my office Gmail account ifoget basword

**Comment by jfullstackdev** [comment]
usually there is your number that Gmail can send a code confirmation, right ?

**Comment by queenofcorgis** [comment]
Hi @goback1 ,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Gmail. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by goback1** [reply]
@goback1 ![IMG_20221019_095907_966.png](https://user-images.githubusercontent.com/73076771/196627795-61ed1416-5a66-4044-9680-b0dece067e46.png)
